src/main/java/docSharing/controller/AbstractController.java

```diff
@@ -115,6 +180,7 @@ public ResponseEntity<Object> relocate(Long newParentId, Long id, Class c) {
      * @return the service we need to use according to what file it is.
      */
     private ServiceInterface convertFromClassToService(Class c) {
+        logger.info("in AbstractController -> convertFromClassToService");
```

Comment:
Personally, I would skip "supportive" functions and would not include logs in them, since their job is to complete the main function's job.
Tell me what you think